tests/integration/conftest.py

```diff
@@ -28,7 +28,7 @@
 logger = trino.logging.get_logger(__name__)
 
 
-TRINO_VERSION = os.environ.get("TRINO_VERSION") or "351"
+TRINO_VERSION = os.environ.get("TRINO_VERSION") or "latest"
```

Comment:
@hashhar that was copied from your PR: https://github.com/trinodb/trino-python-client/pull/128 

Looks like your PR is good to be merged. 
I need to test my change on newer Trino version, since there were some changes in auto-commit.

Reply:
@hashhar Could you share your conclusion about testing version matrix?

Reply:
Thanks for reminder. I've merged the linked PR.

I however wanted to do something similar to how we test the older JDBC drivers against current server - testing a sliding window of versions (which can be a follow-up).